modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -247,3 +247,77 @@ class Configuration(six.with_metaclass(TypeWithDefault, object)):
                "Version of the API: {{version}}\n"\
                "SDK Package Version: {{packageVersion}}".\
                format(env=sys.platform, pyversion=sys.version)
+
+    def get_host_settings(self):
+        """Gets an array of host settings
+
+        :return: An array of host settings
+        """
+        return [
+            {{#servers}}
+            {
+                'url': "{{{url}}}",
+                'description': "{{{description}}}{{^description}}No description provided{{/description}}",
+                {{#variables}}
+                {{#-first}}
+                'variables': {
+                {{/-first}}
+                    '{{{name}}}': {
+                        'description': "{{{description}}}{{^description}}No description provided{{/description}}",
+                        'default_value': "{{{defaultValue}}}",
+                        {{#enumValues}}
+                        {{#-first}}
+                        'enum_values': [
+                        {{/-first}}
+                            "{{{.}}}"{{^-last}},{{/-last}}
+                        {{#-last}}
+                        ]
+                        {{/-last}}
+                        {{/enumValues}}
+                        }{{^-last}},{{/-last}}
+                {{#-last}}
+                    }
+                {{/-last}}
+                {{/variables}}
+            }{{^-last}},{{/-last}}
+            {{/servers}}
+        ]
+
+    def get_host_from_settings(self, index, variables={}):
+        """Gets host URL based on the index and variables
+        :param index: array index of the host settings
+        :param variables: hash of variable and the corresponding value
+        :return: URL based on host settings
+        """
+
+        servers = self.get_host_settings()
+
+        # check array index out of bound
+        if index < 0 or index > len(servers):
```

Comment:
the second expression should be `>=`

Reply:
Good catch. Just pushed out a fix.

Reply:
FYI. I'm also reviewing other clients and filing a fix accordingly if the same issue is found.